src/content-tags-drawer/common/context.js

```diff
@@ -35,3 +35,9 @@ export const ContentTagsDrawerContext = React.createContext({
   closeToast: /** @type{() => void} */ (() => {}),
   setCollapsibleToInitalState: /** @type{() => void} */ (() => {}),
 });
+
+/* istanbul ignore next */
+export const ContentTagsDrawerSheetContext = React.createContext({
+  blockingSheet: /** @type{boolean} */ (false),
+  setBlockingSheet: /** @type{Function} */ (() => {}),
+});
```

Comment:
Additionally, it seems like the value of 'blockingSheet' is just derived from whether tags have been added or not, an actual count is not important.

Reply:
I have added one more layer above, currently it is like this: `ContentTagsDrawerSheet > ContentTagsDrawer > ContentTagsCollapsible`. The drawer is used in:
- `frontend-app-course-authoring` UI: Here `ContentTagsDrawerSheet` is used.
- `edx-platform` UI as iframe: Here `ContentTagsDrawer` is used, not `ContentTagsDrawerSheet`.

I have separated it because `ContentTagsDrawerSheetContext` is `ContentTagsDrawerSheet` logic and is not used in the legacy `edx-platform` screens. This will change in the future, so I'm going to add comments in the code

Reply:
> Additionally, it seems like the value of 'blockingSheet' is just derived from whether tags have been added or not, an actual count is not important.

Updated here: https://github.com/openedx/frontend-app-course-authoring/pull/965/commits/1fe628231dc4717d064726babbb7b76bc53c9b4b

Reply:
Comments added here: https://github.com/openedx/frontend-app-course-authoring/pull/965/commits/cf746b4646e972556e958f62a97360b4eb8d1ee5